src/test/java/no/sikt/nva/scrapers/EmbargoScraperTest.java

```diff
@@ -18,22 +26,41 @@ public class EmbargoScraperTest {
                                           + " precipitation fields with variance consistent interpolation.pdf";
     public static final String DATE = "2023-10-01";
     public static final String TEST_FILE_LOCATION = "src/test/resources/FileEmbargo.txt";
-    public static final String TEST_FILE_LOCATION_V2 = "src/test/resources/FileEmbargoV2.txt";
-
 
     @Test
-    void shouldExtractEmbargoWithPdfFileOnly() {
+    void shouldExtractEmbargoWithPdfFile() {
         var expectedEmbargo = new Embargo(HANDLE, FILENAME, DATE);
-        var actualEmbargo = Objects.requireNonNull(EmbargoScraper.getEmbargoList(new File(TEST_FILE_LOCATION))).get(0);
-        assertThat(actualEmbargo, is(equalTo(expectedEmbargo)));
+        var actualEmbargos =
+            Objects.requireNonNull(EmbargoScraper.getEmbargoList(new File(TEST_FILE_LOCATION))).get(HANDLE);
+        assertThat(actualEmbargos, hasItem(expectedEmbargo));
     }
 
     @Test
-    void shouldExtractEmbargoWithPdfFileOnlyWhen() {
-        var expectedEmbargo = new Embargo(HANDLE, FILENAME, DATE);
-        var actualEmbargo = Objects.requireNonNull(EmbargoScraper.getEmbargoList(new File(TEST_FILE_LOCATION_V2)))
-                                .get(0);
-        assertThat(actualEmbargo, is(equalTo(expectedEmbargo)));
+    void shouldLogNonDetectedEmbargos() {
+        var appender = LogUtils.getTestingAppenderForRootLogger();
+        var embargos = EmbargoScraper.getEmbargoList(new File(TEST_FILE_LOCATION));
+        EmbargoParser.logNonEmbargosDetected(embargos);
+        assertThat(appender.getMessages(), containsString("Embargo file not found: "));
+    }
```

Comment:
Denne logger nå alle "ignorerte filer"
Burde vi logge filer vi ikke ignorerer?

for eksempel:

shouldLogNonDetectedNotIgnoredFiles?

FOLLOWING COLLECTION CONTAINS 1 EMBARGOES.n.s.n.s.EmbargoParser:  
Embargo file not found: Simulated precipitation fields with variance consistent interpolation.pdf.jpg, for handle: https://hdl.handle.net/11250/2683076n.s.n.s.EmbargoParser:  Embargo file not found: license_rdf, for handle: https://hdl.handle.net/11250/2683076n.s.n.s.EmbargoParser:  Embargo file not found: license.txt fields with variance consistent interpolation.pdf.txt, for handle: https://hdl.handle.net/11250/2683076

Eller ønsker vi masse logmeldinger med at vi ignorerer license_rdf filen og license.txt filen?

Reply:
Nice catch. refaktorert slik at det ikke skal komme ut.